save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -96,16 +98,26 @@ class DownloadFilesController(
     /**
      * @param file image to be uploaded
      * @param owner owner name
+     * @param isOrganization flag of organization
      * @return [Mono] with response
      */
     @Suppress("UnsafeCallOnNullableType")
     @PostMapping(value = ["/api/image/upload"], consumes = [MediaType.MULTIPART_FORM_DATA_VALUE])
-    fun uploadImage(@RequestPart("file") file: Mono<FilePart>, @RequestParam owner: String) =
-            additionalToolsFileSystemRepository.saveImage(file, owner).map { imageInfo ->
+    fun uploadImage(
+        @RequestPart("file") file: Mono<FilePart>,
+        @RequestParam owner: String,
+        @RequestParam(required = false) isOrganization: Boolean = true
```

Comment:
Maybe, we should introduce a enum for image kinds?

Reply:
Yes. Added enum and changed logic